invenio_administration/assets/semantic-ui/translations/invenio_administration/scripts/compileCatalog.js

```diff
@@ -1,39 +1,124 @@
 // This file is part of React-Invenio-Deposit
+// Copyright (C) 2021-2024 Graz University of Technology.
+// Copyright (C) 2025 KTH Royal Institute of Technology.
 //
-// Invenio-administration is free software; you can redistribute it and/or modify it
+// Invenio-app-rdm is free software; you can redistribute it and/or modify it
```

Comment:
i would not change this line, although i have seen in the python code of invenio-administration `Invenio is a free software`; `InvenioAdministration is a free software`, `Invenio App RDM is a free software` and if i would search further maybe i would find other lines too. until now i thought this is consistent in a package, but it is not. but i think it is not necessary to change it!